### PR TITLE
patat v0.8.7.0 (new formula)

### PR DESCRIPTION
**Pull request by athas**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/patat.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
+class Patat < Formula
+  desc "Terminal-based presentations using Pandoc"
```
Comment (SMillerDev) [inline Formula/patat.rb:2]: Shouldn't it depend on pandoc then?

Comment (athas) [reply Formula/patat.rb:2]: It uses Pandoc as a Haskell library, so it's essentially compiled into the executable.

**Comment by athas** [comment]
Anything more that has to be done?  The ARM task fails because (I guess) GHC cannot generate ARM binaries yet.

**Comment by athas** [comment]
Could you re-run CI?  I think that was a transient error.

**Pull request closed without merging**